linkerd/proxy/api-resolve/src/metadata.rs

```diff
@@ -1,12 +1,14 @@
 use http::uri::Authority;
-use indexmap::IndexMap;
 use linkerd_tls::client::ServerId;
+use std::collections::BTreeMap;
+
+pub type Labels = BTreeMap<String, String>;
```

Comment:
maybe worth a comment here that `BTreeMap` is used because we want these to be sorted?